components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -89,13 +89,108 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
+
+  {{< figure src="/images/automate/attributes-in-environment.png" alt="Attributes In Environment">}}
+
+#### Delete an Environment
+
+The Chef Infra Server lets you delete environments one at a time. Select **Delete** from the more information (three dots) icon on the far right side of the environment listing that you want to delete, as shown below:
+
+{{< figure src="/images/automate/delete-an-environment.png" alt="Delete an Environment by selecting the more information icon on the right side of an environment list item">}}
+
 ### Data Bags
 
 [Data bags]({{< relref "data_bags" >}}) store global variables as JSON data. Data bags are indexed for searching and can be loaded by a cookbook or accessed during a search.
 
 ### Clients
 
-Chef Infra Clients provide secure API access to the Chef Infra Server.
+Chef Infra Clients provide secure API access to the Chef Infra Server. Chef Infra Server UI lets you:
+
+- Create a client.
+- Search for a specific client from a list of clients.
+- List out all the clients at one place.
+- Reset a client key.
+- Delete a client.
+
+#### Create a Client
+
+To create a new client, select **Create Client**.
+
+{{< figure src="/images/automate/create-client-button.png" alt="Create Client Button">}}
+
+Selecting the **Create Client** button opens a dialog box. The dialog box contains a section to add _Client Name_ and also an option to select whether the client is _Validation Client_ or not. Select **Create** .
+
+{{< figure src="/images/automate/create-client-popup.png" alt="Create Client Dialog Box">}}
+
+Selecting **Create** opens a dialog box which contains the _Private Key_ of that particular client. Select **Download** to download the _Private Key_.
+
+{{< figure src="/images/automate/client-private-key.png" alt="Create Private Key of a Client">}}
+
+#### Search for a Client
+
+Use the **Search Clients** bar to find an client from the list of clients.
+Entering the name of a client in the search bar returns clients matching your search criteria.
+
+{{< figure src="/images/automate/create-client-button.png" alt="Create Client Button">}}
+
+#### Public Key of a Client
+
+To view the _Public Key_ of a specific client, select the client. You will see the _Public Key_ of that client in **Details**.
+
+{{< figure src="/images/automate/client-details-public-key.png" alt="Public Key of Clients">}}
+
+The Chef Infra Server lets you reset the _Public Key_ using the _Reset Key_ option. Selecting the _Reset Key_ option opens an alert stating _The current key will no longer be accepted._
+
+{{< figure src="/images/automate/reset-key-in-client-details.png" alt="Reset Key option of a Client" width="400" height="300">}}
+
+Selecting the _Reset Key_ shown in the above image opens a new dialog box which contains the new _Private Key_ of that client. To download the new _Private Key_, select **Download**.
+
+{{< figure src="/images/automate/reset-public-key-of-a-client.png" alt="Reset Public Key of a Clients">}}
```

Comment:
```suggestion
{{< figure src="/images/automate/reset-public-key-of-a-client.png" alt="Reset Public Key of a Client" width="500">}}
```

Reply:
Let's discuss standards for images in our meeting